### ISSUE TITLE
Edematous chorionic villi

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5681]
**Preferred term label:**
Edematous chorionic villi

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

Near here -- might need to restructure
Abnormality of prenatal development or birth
https://hpo.jax.org/app/browse/term/HP:0001197

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6122977/

**Your nano-attribution (ORCID)**
